code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -346,7 +346,7 @@
 	required_reagents = list("stable_plasma" = 1, "radium" = 1, "phosphorus" = 1)
 
 /datum/chemical_reaction/pyrosium/on_reaction(datum/reagents/holder, created_volume)
-	holder.chem_temp = 20 // also cools the fuck down
+	holder.chem_temp += 500 // also heats the fuck up AGAIN
```

Comment:
Meh pyrosium is pretty useless anyway, any decent chemist who is actually using it is going to precisely heat/cool the chems using the heater anyway. A fire chem cooling the container to subzero when made just seems like an oversight to me.